### PR TITLE
Dichotomy keymap by alairock

### PR DESCRIPTION
**Pull request by alairock**
Adding a keymap for the Dichotemy keyboard. It will now more closely resemble my keymap used on the minidox

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by this PR

* 

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dichotomy/keymaps/alairock/keymap.c [hunk 1]
```diff
@@ -0,0 +1,205 @@
+// this is the style you want to emulate.
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+
+#include "dichotomy.h"
```
Comment (noroadsleft) [inline keyboards/dichotomy/keymaps/alairock/keymap.c]: This should actually be:

```suggestion
#include QMK_KEYBOARD_H
```

Not your fault really; the default keymap needs to be updated as well.

File: keyboards/dichotomy/keymaps/alairock/keymap.c [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+  MS_BTN1,
+  MS_BTN2,
+  MS_BTN3
+};
+
+#define RED_BRIGHTNESS 3
+#define GREEN_BRIGHTNESS 2
+#define BLUE_BRIGHTNESS 2
+
+// Fillers to make layering more clear
```
Comment (noroadsleft) [inline keyboards/dichotomy/keymaps/alairock/keymap.c]: Lines 33-35 can be deleted; QMK's build script includes these definitions in every keymap.

https://github.com/qmk/qmk_firmware/blob/7c63c0a95075f6fb4dd94d020ceb5816e6bcca2c/quantum/quantum_keycodes.h#L29-L31

File: keyboards/dichotomy/keymaps/alairock/keymap.c [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+  _______, _______, _______, _______, _______, _______,        _______, MS_BTN1, MS_BTN2, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,        KC_VOLD, KC_END,  KC_PGDN, _______, _______, _______,
+  							 _______, _______, _______,        _______, KC_UP,   _______,
+					_______, _______, _______, _______,        KC_LEFT, KC_DOWN, KC_RGHT, _______
+)
+
+}; // end keymaps block
+
+
+const uint16_t PROGMEM fn_actions[] = {
```
Comment (noroadsleft) [inline keyboards/dichotomy/keymaps/alairock/keymap.c]: Lines 83-85 should be deleted.

**Comment by alairock** [comment]
Fixed suggestions

**Comment by drashna** [comment]
Thanks!

**Pull request merged**